### DISCUSSION TITLE
发点原神照片

### DISCUSSION THREAD

**Discussion by Hjhnull** [share121/inter-knot#5440] [原神]
![Screenshot_20240829_095409_com.m4399.gamecenter.jpg](https://github.com/user-attachments/assets/76d9100d-aa80-40f2-b0de-e491ed82beae)

![Screenshot_20240829_100339_com.m4399.gamecenter.jpg](https://github.com/user-attachments/assets/aee29064-7a0b-4280-9f4b-6a8f865ae4c6)

![202482415324589.jpg](https://github.com/user-attachments/assets/621aa6bc-e21c-44b3-8df4-c3fc4ed9d1c4)

![17244787385999da1a8e36acd7e326eac0c5a70c25e2e7c9e1b6a0149d54d17901576923a5fd1.0.jpg](https://github.com/user-attachments/assets/fae0ea25-0d3e-4159-b4cf-e6cadbbd56bd)

![Screenshot_20240829_101016_com.m4399.gamecenter.jpg](https://github.com/user-attachments/assets/7d29f28e-4c10-4cd0-9fb5-3ab09325abe7)

![Screenshot_20240825_151614_com.miHoYo.cloudgames.ys.jpg](https://github.com/user-attachments/assets/84a195df-3e2b-4f7f-ae17-0baaa310a7ee)

![202482514599394.jpg](https://github.com/user-attachments/assets/a4571b51-6c17-4411-8e79-d0ca8c5697c1)

![Screenshot_20240829_110333_com.m4399.gamecenter.jpg](https://github.com/user-attachments/assets/b7f82fbb-2c73-4dc8-b87d-87c4c8c7a9c0)

![image_download_1724405144173.jpg](https://github.com/user-attachments/assets/57bbe012-3826-4343-9725-f07c9fd26729)

**Comment by abyssal-sea-c** [comment]
没发出来喵

**Reply by Hjhnull** [reply]
我一个个发

**Reply by Hjhnull** [reply]
或者到github里看
